controller/src/beerocks/master/tasks/dynamic_channel_selection_task.cpp

```diff
@@ -103,19 +106,28 @@ void dynamic_channel_selection_task::work()
         if (m_is_single_scan_pending) {
             m_is_single_scan         = true;
             m_is_single_scan_pending = false;
```

Comment:
(nitpick ) I know its not part of this PR but `m_is_single_scan_pending` and `m_is_single_scan_pending` confused me .
i prefer `m_is_manual_scan` and also would add a comment in code why do we need to treat manual scan differently from continues one .

Reply:
will add it as a separate hotfix, but for discussion, we use the "single" terminology across the entire DCS feature implementation - why should the task name it differently?
we can change it to `m_handling_single_scan_request` if it would be more clear.

secondly - the `m_is_single_scan_pending` is a flag that indicates that a single scan request was made and haven't been treated yet.
the `m_is_single_scan` is used through the entire task to determine which scan params to read and which scan results to update. where do you think a comment regarding this should be added?

Reply:
> will add it as a separate hotfix, but for discussion, we use the "single" terminology across the entire DCS feature implementation - why should the task name it differently?
> we can change it to `m_handling_single_scan_request` if it would be more clear.
>
In that case just adding "request" would do 
`m_single_scan_request` 
`m_single_scan_request`

> 
> secondly - the `m_is_single_scan_pending` is a flag that indicates that a single scan request was made and haven't been treated yet.
> the `m_is_single_scan` is used through the entire task to determine which scan params to read and which scan results to update. where do you think a comment regarding this should be added?

i would add it to the header file